lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
I just thought about it and it felt like a very specific case of our own builder when running our own test suite. I am not sure it will add any value to anyone else to change the builder API.